### ISSUE TITLE
Check on progress with adding alt text to main site

### ISSUE THREAD

**Issue by soshwosh** [lsst-epo/rubin-obs-client#1141]
As part of an effort to tidy up some our accessibility issues on the main site, [some initial discovery](https://github.com/lsst-epo/rubin-obs-client/issues/1133) revealed that some photos in the Craft CMS Asset library are missing alt text. A list of the highest priority images can be found [here](https://docs.google.com/document/d/1HttDXYd1EkSgIuJ2Ux7tc7gjv2XhaEF2fUvB1Kf7-sw/edit?tab=t.bfg9s2nevwbn). Follow up on this effort and re-run the Lighthouse audits on the corresponding pages to confirm increased adherence to industry accessibility standards.
